operations/http.go

```diff
@@ -293,12 +293,13 @@ func (ac *legacyClient) DeletePatchModule(patchId, module string) error {
 }
 
 // UpdatePatchModule makes a request to the API server to set a module patch on the given patch ID.
-func (ac *legacyClient) UpdatePatchModule(patchId, module, patch, base string) error {
+func (ac *legacyClient) UpdatePatchModule(patchId, module, patch, base, message string) error {
```

Comment:
it wouldn't be a bad thing if there was a struct that you passed to this to get ~keyed args.

Reply:
done.